flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1030,6 +1030,13 @@ void createTemporarySystemFunction(
      */
     boolean dropTemporaryTable(String path);
 
+    /**
+     * Drops a table in a given fully qualified path.
```

Comment:
The path doesn't need to be fully qualified. We should copy existing explanation to not make mistakes.
Also we need information about temporary vs permant objects. I guess it drops only permanent tables, not temporary ones. If temporary still exists it throws an exception?